README.md

```diff
@@ -189,16 +189,25 @@ governance process.  A complete list of `ilk`s for a deployment may be gleaned f
 
 ## Gas price strategy
 
-Auction keeper can be configured to use several API sources for retrieving gas prices:  
-    - **Ethgasstation** if a key is passed as `--ethgasstation-api-key` (e.g. `--ethgasstation-api-key MY_API_KEY`)  
-    - **Etherchain.org** if keeper started with `--etherchain-gas-price` switch  
-    - **POANetwork** if keeper started with `--poanetwork-gas-price` switch. An alternate URL can be passed as `--poanetwork-url`,
+Auction keeper can use one of several sources for the initial gas price of a transaction:  
+ * **Ethgasstation** if a key is passed as `--ethgasstation-api-key` (e.g. `--ethgasstation-api-key MY_API_KEY`)  
+ * **Etherchain.org** if keeper started with `--etherchain-gas-price` switch  
+ * **POANetwork** if keeper started with `--poanetwork-gas-price` switch. An alternate URL can be passed as `--poanetwork-url`,
     that is useful when server hosted locally (e.g. `--poanetwork-url http://localhost:8000`)  
-
-If no gas price type specified or gas price API not accessible then keeper will apply an increased gas price, starting
-with a value of 5 GWEI and increased by 10 GWEI each minute, up to 100 GWEI.
-
-Note: this gas strategy is used by keeper in all interactions with chain but when sending a bid (which is provided by model)
+ * The `--fixed-gas-price` switch allows specifying a **fixed** initial price in Gwei (e.g. `--fixed-gas-price 12.4`) 
+ 
+When using an API source for initial gas price, `--gas-initial-multiplier` (default `1.0`, or 100%) tunes the initial 
+value provided by the API.  This is ignored when using `--fixed-gas-price` and when no strategy is chosen.  If no 
+initial gas source is configured, or the gas price API produces no result, then the keeper will start with a price of 
+10 Gwei.
+
+Auction keeper periodically attempts to increase gas price when transactions are queueing.  Every 30 seconds, a 
+transaction's gas price will be multiplied by `--gas-initial-multiplier` (default `2.25`, or 225%) until it is mined or 
```

Comment:
Typo: should be `--gas-reactive-multiplier` instead of `--gas-initial-multiplier`.

We should consider mentioning Parity's (and Geth's?) minimum increase (`1.125`) to replace a transaction. This could be done either in the keeper README or as a floor (`assert (coefficient > 1.125)`) in the AuctionKeeper's `__init__()`.

Pymaker mentions this in the `GeometricGasPrice` class but does prevent a low coefficient from being used - the class only ensures that the argument is over 1: `assert (coefficient > 1)`

Reply:
Great feedback; I pushed an update to `README` which I believe addresses these concerns.